src/renderer/kube-object.store.ts

```diff
@@ -345,8 +345,8 @@ export abstract class KubeObjectStore<T extends KubeObject = any> extends ItemSt
       if (!this.isLoaded || error instanceof DOMException) return;
 
       if (error instanceof Response) {
-        if (error.status === 404) {
-          // api has gone, let's not retry
+        if (error.status === 404 || error.status === 401) {
```

Comment:
This worries me a bit because we don't recover from 401 (if credentials are refreshed after 401 happens).

Reply:
This doesn't recover I agree. However, if the user moves to a different page then the credentials will be tried again. The thing is if the credentials are ENV bound then we would have to sync that  before trying again.

So we would have to sync from ENV every so often (I think) as well as retrying this. That is slightly more complicated though.